### PR TITLE
[PLAYER-3816] OoyalaIMA manager should be initialized after embed code has been set up

### PR DESCRIPTION
**Pull request by AlinaVoronkovaEpam**
_Issue description_
In `fetchCuePoint()` method, `_player.getDuration()` returns 1 instead of a real duration, because `player` hasn't completely initialized yet.

_Solution_
Set `embed code` before OoyalaIMA manager initialization.

### CODE REVIEW

File: ExoPlayerSampleApp/app/src/main/java/com/ooyala/sample/players/PreconfiguredIMAPlayerActivity.java [hunk 1]
```diff
@@ -48,12 +48,12 @@ void completePlayerSetup(boolean asked) {
...
 
-      @SuppressWarnings("unused")
-      OoyalaIMAManager imaManager = new OoyalaIMAManager(player, skinLayout);
-
       if (player.setEmbedCode(embedCode)) {
         //player.play();
       }
+
+      @SuppressWarnings("unused")
+      OoyalaIMAManager imaManager = new OoyalaIMAManager(player, skinLayout);
```
Comment (pilievOoyala) [inline ExoPlayerSampleApp/app/src/main/java/com/ooyala/sample/players/PreconfiguredIMAPlayerActivity.java:56]: Shouldn't this come before the setEmbedCode statement like in two activites above this file?

**Comment by pilievOoyala** [comment]
I am confused with this checkin. Half of the changes make the setEmbedCode come after IMA init and the other half make it before.  If things are order dependent please put in comments about that. Reference the ticket number and what the order is that is needed.

**Comment by brol1dev** [comment]
@pilievOoyala I saw that all the changes are correct. Set embed codes happens before creating the `OoyalaIMAManager`.

**Review by brol1dev — APPROVED** [review]
LGTM

**Pull request merged**